### ISSUE TITLE
my lesson1 result is much worse than original

### ISSUE THREAD

**Issue by zack1030** [zack1030/fast.ai#1]
mine
loss: 0.4553 - acc: 0.8648 - val_loss: 0.2205 - val_acc: 0.9150
original
loss: 0.1116 - acc: 0.9696 - val_loss: 0.0528 - val_acc: 0.9835

**Comment by zack1030** [comment]
1. Theano and TensorFlow implement Conv2D in different ways, vgg16.h5 needs to be converted for TensorFlow.
https://github.com/fchollet/keras/wiki/Converting-convolution-kernels-from-Theano-to-TensorFlow-and-vice-versa
2. there are some issue in preprocess function (vgg16.py)
* the video says that we have to change channels' order, but the original code only rotate the image
* the vgg_mean in the original code is in RGB order
